ThunderRequest/BackgroundSessionController.swift

```diff
@@ -27,15 +27,21 @@ public class BackgroundRequestController: NSObject, URLSessionDelegate, URLSessi
     
     private var urlSession: URLSession?
     
+    private let readData: Bool
```

Comment:
Not sure this is needed as in instance property

Reply:
We need it to check on line `66` if we're going to read the file into `Data` :D

Reply:
Oh sorry Github was hiding code an thought it was part of the `init`